platform/linuxbsd/detect.py

```diff
@@ -309,6 +360,14 @@ def configure(env: "Environment"):
     )
 
     if env["x11"]:
+        if not env["use_sowrap"]:
+            env.ParseConfig("pkg-config x11 --cflags --libs")
+            env.ParseConfig("pkg-config xcursor --cflags --libs")
+            env.ParseConfig("pkg-config xinerama --cflags --libs")
+            env.ParseConfig("pkg-config xext --cflags --libs")
+            env.ParseConfig("pkg-config xrandr --cflags --libs")
+            env.ParseConfig("pkg-config xrender --cflags --libs")
+            env.ParseConfig("pkg-config xi --cflags --libs")
```

Comment:
That's going to add a lot of boilerplate but I guess we should check that those exist too, and `sys.exit()` if any is missing.

Reply:
Added checks for X11 libs and warnings for the rest.